lib/oxidized/model/model.rb

```diff
@@ -124,14 +124,14 @@ def cmd(string, &block)
 
       out = out.b unless Oxidized.config.input.utf8_encoded?
       self.class.cmds[:all].each do |all_block|
-        out = instance_exec Oxidized::String.new(out), string, &all_block
```

Comment:
We don't need to recast string into string, just 'out' is fine.

Reply:
done